roles/openstack-network/handlers/main.yml

```diff
@@ -2,3 +2,14 @@
 - name: ifup neutron external interface
   command: ifup {{ neutron_external_interface }}
   ignore_errors: yes
+
+- name: enable ip forwarding
+  sysctl: name="net.ipv4.conf.all.forwarding" value=1 sysctl_set=yes state=present reload=yes
+ 
+- name: enable ip masq
```

Comment:
I don't think we want this here?   AFAIK the masquerade should only be necessary to support CI so can just be in playbooks/ci-full-centos/tasks/pre-deploy.yml